plugin/contrib/mesos/pkg/scheduler/service/service.go

```diff
@@ -68,6 +72,8 @@ const (
 	defaultReconcileInterval = 300    // 5m default task reconciliation interval
 	defaultReconcileCooldown = 15 * time.Second
 	defaultFrameworkName     = "Kubernetes"
+	containerCpus            = 0.25 // initial CPU allocated for executor
```

Comment:
these are defined somewhere else too (podtask?) can we just have the cpus and mem consts in one place, perhaps pkg/scheduler/podtask/leaky.go? we'll get rid of them once we implement proper resource accounting and parent cgroup support